.ci/packaging.groovy

```diff
@@ -154,6 +155,30 @@ pipeline {
                   prepareE2ETestForPackage("${BEATS_FOLDER}")
                 }
               }
+              stage('Package Docker images for linux/arm64'){
+                agent { label 'arm' }
+                options { skipDefaultCheckout() }
+                when {
+                  beforeAgent true
+                  expression {
+                    return params.arm
+                  }
+                }
+                environment {
+                  HOME = "${env.WORKSPACE}"
+                  PACKAGES = "docker"
```

Comment:
```suggestion
                  PACKAGES = "docker"
                  GOARCH = "arm64"
```

Reply:
This is not required anymore https://github.com/elastic/apm-pipeline-library/pull/928 is now in charge to set the arch for installing go and so on in case GOARCH was not set.